src/Altinn.AccessManagement.Persistence/PolicyRepository.cs

```diff
@@ -147,39 +152,48 @@ public async Task<Response> DeletePolicyVersionAsync(string filepath, string ver
             }
             catch (RequestFailedException ex)
             {
+                string errorMsg;
```

Comment:
errorMsg can be set using a ternary, then you don't need to define it, and then assign a value to the variable?

var errormsg = ex.Status == (int)HttpStatusCode.Forbidden && ex.ErrorCode == "OperationNotAllowedOnRootBlob" ?
   $"Failed to delete version {version} of policy file at {filepath}. Not allowed to delete current version." :
   $"Failed to delete version {version} of policy file at {filepath}. RequestFailedException";